src/doom/g_game.c

```diff
@@ -1319,6 +1319,15 @@ int cpars[32] =
     240,150,180,150,150,300,330,420,300,180,	// 21-30
     120,30					// 31-32
 };
+
+// Chex Quest Par Times
+int qpars[4][10] = 
```

Comment:
This should be `static const`. So should `pars` and `cpars` above but that should be a separate commit (and PR). Can dehacked change these? Based on a quick source inspection it would seem not in the current implementation.